controller/parser/multicast_request_parser.go

```diff
@@ -87,8 +87,11 @@ func resolveArgsByServer(serverIDs []int64, allServerArgs []interface{}) (map[in
 
 func removeNamespace(method string) string {
 	parts := strings.Split(method, ".")
-	slice := parts[1:len(parts)]
-	return strings.Join(slice, ".")
+	if len(parts) > 1 {
```

Comment:
shouldn't we fail with an error here if this has no "multicast" namespace?

Reply:
See the commit message:

Namespaces are optional according to the XMLRPC standard

http://xmlrpc.com/spec.md

(and actually are Uyuni-specific, as dots per se have no meaning in
XMLRPC method names)

Reply:
True. But in this app, multicasting is executed if and only if the "multicast" namespace is present. So I guess if we got here and there's no "multicast" namespace something went wrong for sure.

Reply:
Are you suggesting the test is wrong? Currently it only calls `"method"`